js/ui/scheduler/ui.scheduler.appointments.strategy.vertical.js

```diff
@@ -238,9 +211,44 @@ var VerticalRenderingStrategy = BaseAppointmentsStrategy.inherit({
             result = allDayCondition ? allDayCondition : this._rowCondition(a, b);
 
         return this._fixUnstableSorting(result, a, b);
-    }
+    },
 
+    _getDynamicAppointmentCountPerCell: function() {
+        return 3;
```

Comment:
Can we use const here? `var APPT_COUNT_PER_CELL = 3;` This value is used in the `_getDefaultRatio` method too.